make/common/JdkNativeCompilation.gmk

```diff
@@ -291,6 +291,10 @@ define SetupJdkExecutableBody
       -I$(TOPDIR)/src/java.base/windows/native/common \
       $$($1_EXTRA_RCFLAGS)
 
+  ifneq ($$($1_HEADERS_FROM_SRC), false)
```

Comment:
I note that this parameter in SetupJdkLibrary is documented in the comment above the definition, but you haven't added any of the new parameters to SetupJdkExecutable to the comment for that macro. Should we, or is the intention to rework this file anyway?

Reply:
The intention is to unify SetupJdkLibrary and SetupJdkExecutable, and clean up this file at the same time. I have tried to keep it up to date meanwhile, but I have missed it here. So for me it's either or; I can fix it now, or later.